src/element-style-observer.js

```diff
@@ -5,13 +5,14 @@ import { toArray, wait, getTimesFor } from "./util.js";
 import RenderedObserver from "./rendered-observer.js";
 
 // Start bug detection early if we have a DOM
+let allowDiscrete = "";
 if (globalThis.document) {
-	Bugs.detect();
-}
+	Bugs.detectAll();
```

Comment:
Is this intentionally imported as `Bugs` instead of `bugs`?

Reply:
I was trying to minimize changes, and since it's a default import it doesn't matter. But `bugs` would probably be better indeed!